frappe/utils/backups.py

```diff
@@ -82,7 +82,7 @@ def get_backup(self, older_than=24, ignore_files=False, force=False):
 	def set_backup_file_name(self):
 		#Generate a random name using today's date and a 8 digit random number
 		for_db = self.todays_date + "-" + self.site_slug + "-database.sql.gz"
-		for_public_files = self.todays_date + "-" + self.site_slug + "-files.tar"
+		for_public_files = self.todays_date + "-" + self.site_slug + "-public-files.tar"
```

Comment:
Instead of changing the name of the files, could we just fix `get_latest_backup_file` instead? This breaks existing behavior since it's also ported to v12.

Reply:
what is it breaking?